### PR TITLE
Feature/fis 320 severity view

### PR DESCRIPTION
**Pull request by nikiwycherley**
Severity View Tab

https://github.com/DEFRA/fws-tests/blob/master/features/ManagementConsole/Release1/Screens/SeverityReport.feature

Added Severity View Tab to show totals of each severity and list of each Target Area with and active alert.

### CODE REVIEW

File: test/routes/index-happy.js [hunk 1]
```diff
@@ -242,6 +242,12 @@ lab.experiment(('All basic routes'), () => {
       strategy: 'azure-legacy',
       credentials: postLoginCredentials
     }
+  }, {
+    url: '/severity-view',
+    code: 200,
+    text: [
+      'Severity view</h2>'
```
Comment (davidjamesstone) [inline test/routes/index-happy.js]: `</h2>` ? No opening tag.

File: server/routes/severity-view.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  options: {
+    ext: {
+      onPreAuth: {
+        method: (request, h) => {
+          // Set login redirect cookie
+          h.state('login-redirect', request.path)
+          return h.continue
+        }
+      }
+    },
```
Comment (davidjamesstone) [inline server/routes/severity-view.js]: This is no longer necessary and should be removed. See https://github.com/DEFRA/fws-app/commit/5a74b601e0b806bbc9ef95c3ebb72322a74b0aa0

File: server/models/severity-view.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        classes: 'govuk-table__header center',
+        attributes: { valign: 'center' }
+      },
+      {
+        text: 'Total',
+        classes: 'govuk-table__header center',
+        attributes: { valign: 'center' }
+      },
+      {
+        text: 'Owner Area',
```
[the remark this answers is not in the document]
Comment (nikiwycherley) [reply server/models/severity-view.js]: changes made

File: server/routes/severity-view.js (hunk 2 above)
Comment (nikiwycherley) [reply server/routes/severity-view.js]: changes made

File: test/routes/index-happy.js (hunk 1 above)
Comment (nikiwycherley) [reply test/routes/index-happy.js]: left the same as it matches the pattern of other unit tests above.

**Pull request merged**